test/test_torch.py

```diff
@@ -6169,12 +6169,12 @@ def test_neg(self, device):
 
         for dtype in float_types + int_types:
             if dtype in float_types:
-                a = torch.randn(100, 90).type(dtype).to(device)
+                a = torch.randn(100, 90).to(device=device, dtype=dtype)
```

Comment:
nit: this could have been all merged into `torch.randn(100, 90, device=device, dtype=dtype)` no?
That would be a (minor :D ) perf improvement no?

Reply:
I vaguely remember that some dtypes doesn't support `rand` APIs in the past but that might be rare :P

Reply:
Ho ok. Well this is fine then :)